tests/main/views/test_direct_award.py

```diff
@@ -64,6 +65,22 @@ def setup_class(cls):
         data_api_client.find_direct_award_projects = mock.Mock()
         data_api_client.find_direct_award_projects.return_value = cls._get_direct_award_project_list_fixture()
 
+    def test_renders_saved_search_overview(self):
+        self.login_as_buyer()
+        res = self.client.get(self.SAVE_SEARCH_OVERVIEW_URL)
+        assert res.status_code == 200
+
+        result_html = res.get_data(as_text=True)
+        assert "Your saved searches" in result_html
+        assert "Searching" in result_html
+        assert "Search ended" in result_html
+
+        doc = html.fromstring(result_html)
+        open_date = "Sunday 17 September 2017 at 3:33pm GMT"
+        closed_date = "Tuesday 10 October 2017 at 4:55pm GMT"
+        assert doc.xpath('//*[@id="content"]/div/div/table[1]/tbody/tr[1]/td[2]/span')[0].text == open_date
+        assert doc.xpath('//*[@id="content"]/div/div/table[2]/tbody/tr[1]/td[2]/span')[0].text == closed_date
```

Comment:
It might be good to group the first of these two asserts under the 'searching' assert above, and the second under the 'search ended' assert - to more clearly represent associations.

I don't know if such a specific xpath assertion is good or bad. It feels a bit fragile. But it's probably fine.

Reply:
the path gets the first row of the tables so I would leave them as they are I think.